src/collections/_documentation/development/contribute/environment.md

```diff
@@ -11,38 +11,114 @@ Follow [the Docker file in the repository](https://github.com/getsentry/sentry/b
 
 ### Macintosh OS X
 
-To get started, fork the repo at [https://github.com/getsentry/sentry](https://github.com/getsentry/sentry) and clone it:
+To get started, fork the repo at [https://github.com/getsentry/sentry](https://github.com/getsentry/sentry). Clone your forked repo and go into it:
 
 ```bash
 git clone https://github.com/<your github username>/sentry.git
 cd sentry
 ```
 
-Install [Homebrew](http://brew.sh), if you haven’t already. Run `brew bundle` to install the various system packages as listed in sentry's `Brewfile`. This will install, among other things, Python 2 and docker.
+{% capture __alert_content -%}
+It's important that you remain in the repo's root directory for the rest of the setup! Also, please do the following steps in the exact order they're shown.
+{%- endcapture -%}{%- include components/alert.html
+  title="Note"
+  content=__alert_content
+  level="warning"
+%}
+
+
+### Macintosh OS X - System Dependencies
+
+Install [Homebrew](http://brew.sh), and then run `brew bundle` to install the various system packages as listed in sentry's `Brewfile`.
```

Comment:
Is there no way to automate this stuff? Can we not put `/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"` in another script that takes care of everything automatically, including the initial start of Docker?

Reply:
Yes, it can always be automated. But the docs need to be fixed _now_.